### DISCUSSION TITLE
event submission rejected with_reason: ProjectId

### DISCUSSION THREAD

**Discussion by ozgurhanurek** [getsentry/support#101] [Support] [answered]
### Is there an existing issue for this?

- [X] I have checked for existing issues https://github.com/getsentry/sentry-javascript/issues
- [X] I have reviewed the documentation https://docs.sentry.io/
- [X] I am using the latest SDK release https://github.com/getsentry/sentry-javascript/releases

### How do you use Sentry?

Sentry Saas (sentry.io)

### Which package are you using?

@sentry/react

### SDK Version

7.13.0

### Framework Version

7.13.0

### Steps to Reproduce

We've been using Sentry for a while but didn't really utilize it for our React frontend. Now that I'm taking a closer look at it, I'm seeing some issues.

1. I'm setting up Sentry on my local server (`create-react-app` with `react-app-rewired`) according to the guide at https://docs.sentry.io/platforms/javascript/guides/react/
2. I'm trying to throw some errors by either defining a button `onClick` such as the one below, or directly throwing an error in the React component.
```
<button
  type="button"
  onClick={() => {
    throw new Error('CUSTOM SENTRY ERROR');
  }}
>
  Throw error
</button>
```

3. I've failed to reliably send errors to Sentry. I also tried it by first creating another project within my team and then by creating a new project on a new trial account.
4. As far as I can see, this is not isolated to my local server and we are seeing the same problem on production.

### Expected Result

To observe the errors I'm throwing on Sentry issues.

### Actual Result

Most of the time Sentry request are failing with code 403 with the error message event `submission rejected with_reason: ProjectId` which means I don't see the errors on Sentry issues. There are times when the error succeeds which returns an id value:
<img width="603" alt="Screen Shot 2022-09-26 at 17 42 40" src="https://user-images.githubusercontent.com/10331239/192306099-dba42e4b-8a3b-4dfb-ba16-fa6c4336847d.png">

<img width="715" alt="Screen Shot 2022-09-26 at 17 43 18" src="https://user-images.githubusercontent.com/10331239/192306314-79e5e195-0d54-43f9-87a6-95c4e6ff3650.png">

**Comment by souredoutlook** [comment, marked as answer]
Hey @ozgurhanurek - this typically means your DSN is set incorrectly. 

The Sentry DSN [conforms the the format below](https://docs.sentry.io/product/sentry-basics/dsn-explainer/#the-parts-of-the-dsn):

```
{PROTOCOL}://{PUBLIC_KEY}:{SECRET_KEY}@{HOST}{PATH}/{PROJECT_ID}
```

If you're using the SAAS version of Sentry this will typically be `https://{PUBLIC_KEY}@o{ORG_ID}.ingest.sentry.io/{PROJECT_ID}`. If the public key, organization ID, or project ID are not correct, you'll see this error.

The quickest way to check if you're using the correct key is to go to the project settings page for the relevant project:

>https://sentry.io/settings/{org_slug}/projects/{project_slug}/keys/

**Reply by ozgurhanurek** [reply]
Hey @souredoutlook, thanks a lot for your quick response. I did more digging after this and eventually found out that GTM was injecting another Sentry initialization script tag with a different DSN which was causing the errors to go somewhere else entirely. Appreciate your help!
